test/active_model_serializers/test/serializer_test.rb

```diff
@@ -0,0 +1,71 @@
+require 'test_helper'
+require 'rails-controller-testing' if Rails::VERSION::MAJOR >= 5
```

Comment:
@bf4 Yeah I know about the Rails decision removing this test helpers.
But I am not sure if we can remove this test, I think this was added to test the notification was not overwrite, based on the description of the test.

This is the last version of the file https://github.com/rails-api/active_model_serializers/blob/610aeb2e9297fa31b8d561f0be9a4597f0258f8c/test/integration/action_controller/serialization_test_case_test.rb#L67 I found.